build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -505,7 +504,13 @@ def __str__(self):
 
   @staticmethod
   def from_json_object(json_object: Dict[str, Any]):
-    return CompilationInfo(**json_object)
+    return CompilationInfo(
```

Comment:
It would be nice to have a less repetitive way for tuplifying all the list inputs. Maybe when declaring the dataclass you could use a descriptor? https://docs.python.org/3/library/dataclasses.html#descriptor-typed-fields. I think you could also use InitVar, but that would be really repetitive

(this is for later though)